all-instructions/index.html

```diff
@@ -1,53 +0,0 @@
----
```

Comment:
The docker build fails when it tries to copy this deleted directory https://github.com/certbot/website/blob/f1c9566c5b60f7e71d78c631fe19916f64a5b79d/Dockerfile#L53

Reply:
Thanks. I'll make a follow-up PR to handle this with docker-ignore. I didn't want to jam too much (more) into this one.